tensorflow/core/ops/string_ops.cc

```diff
@@ -80,6 +80,13 @@ REGISTER_OP("StringToHashBucketFast")
     .Attr("num_buckets: int >= 1")
     .SetShapeFn(shape_inference::UnchangedShape);
 
+REGISTER_OP("_TensorToHashBucketFast")
+    .Input("input: T")
+    .Output("output: int64")
+    .Attr("T: {int8, uint8, int16, uint16, int32, uint32, int64, uint64}")
+    .Attr("num_buckets: int >= 1")
+    .SetShapeFn(shape_inference::UnchangedShape);
```

Comment:
Please add a `.Doc` (I know this file does not do a good jobs at documentation, but IMO at minimum we should state that the op is a composition of converting the Tensor to a string and then calling `StringToHashBucketFast`).

Reply:
Done.